aten/src/ATen/native/native_functions.yaml

```diff
@@ -5833,9 +5833,13 @@
   dispatch:
     CUDA: _thnn_fused_lstm_cell_cuda
 
-- func: _thnn_fused_lstm_cell_backward(Tensor? grad_hy, Tensor? grad_cy, Tensor cx, Tensor cy, Tensor workspace, bool has_bias) -> (Tensor, Tensor, Tensor, Tensor, Tensor)
+# NB: The composite version of this function below is a simple wrapper that duplicates some of the outputs
+#     It is necessary to avoid triggering TensorImpl use count checks in debug mode
+- func: _thnn_fused_lstm_cell_backward_impl(Tensor? grad_hy, Tensor? grad_cy, Tensor cx, Tensor cy, Tensor workspace, bool has_bias) -> (Tensor, Tensor, Tensor)
   dispatch:
-    CUDA: _thnn_fused_lstm_cell_backward_cuda
+    CUDA: _thnn_fused_lstm_cell_backward_cuda_impl
+
+- func: _thnn_fused_lstm_cell_backward(Tensor? grad_hy, Tensor? grad_cy, Tensor cx, Tensor cy, Tensor workspace, bool has_bias) -> (Tensor, Tensor, Tensor, Tensor, Tensor)
```

Comment:
nit: we might want to have a comment here mentioning that this function is NOT differentiable. Also should we have a this in the impl `TORCH_INTERNAL_ASSERT(!GradMode::enabled());` ?

Reply:
Hmm, don't we already have the autograd not implemented kernel for the case when someone actually tries to double backward on this node? In theory someone could have `create_graph=True` but only `.grad` part of the graph.

Reply:
But then this wouldn't be called?